esx_service/vmdk_ops_test.py

```diff
@@ -479,7 +476,7 @@ def setUp(self):
         si = vmdk_ops.get_si()
         error, self.vm = create_vm(si=si, 
                                    vm_name=self.vm_name, 
-                                   datastore=self.datastore_name)
+                                   vm_path=self.datastore_name)
```

Comment:
this is still weird... why self.datastore_name actually keep the vm_path ? 
I'll approve to unbock , but please investigate as a separate item